Buildings/Fluid/CHPs/OrganicRankine/BaseClasses/FixedEvaporating.mo

```diff
@@ -0,0 +1,204 @@
+        transformation(extent={{100,20},{140,60}}),  iconTransformation(extent={{100,30},
+            {120,50}})));
+  Modelica.Blocks.Interfaces.RealOutput PPum(
+    final quantity="Power",
+    final unit="W")
+    "Electrical power consumption of the pump" annotation (Placement(
+        transformation(extent={{100,-60},{140,-20}}),iconTransformation(extent={
+            {100,-50},{120,-30}})));
+  Modelica.Blocks.Interfaces.BooleanOutput on_actual = ena and hys.y
+    "Actual on off status of the cycle" annotation (Placement(transformation(
+          extent={{100,-20},{140,20}}), iconTransformation(extent={{100,-10},{120,
+            10}})));
+
+  Modelica.Units.SI.ThermodynamicTemperature THotOut(
+    start = TWorEva + dTPinEva_set)
+    "Outgoing temperature of the evaporator hot fluid";
+  Modelica.Units.SI.ThermodynamicTemperature THotPin(
+    start = TWorEva + dTPinEva_set)
+    "Hot fluid temperature at pinch point";
+  Modelica.Units.SI.TemperatureDifference dTPinEva(start = dTPinEva_set)
+    "Pinch point temperature difference of evaporator";
+  Modelica.Units.SI.ThermodynamicTemperature TWorCon
+    "Working fluid condensing temperature";
+  Modelica.Units.SI.ThermodynamicTemperature TColOut
+    "Fluid temperature out of the condenser";
+  Modelica.Units.SI.ThermodynamicTemperature TColPin(
+    start = 300)
+    "Cold fluid temperature at pinch point";
+  Modelica.Units.SI.MassFlowRate mWor_flow
+    "Mass flow rate of the working fluid"
+    annotation (Dialog(group="Cycle"));
+  Modelica.Blocks.Logical.Hysteresis hys(
+    uLow = mWor_flow_min,
+    uHigh = mWor_flow_min + mWor_flow_hysteresis,
+    u = mWor_flow_internal)
+    "Hysteresis for turning off cycle when working fluid flow too low";
+
+protected
+  Modelica.Units.SI.MassFlowRate mWor_flow_internal(
+    start = (mWor_flow_max + mWor_flow_min) / 2)
+    "Working fluid flow rate, intermediate variable";
+  Modelica.Units.SI.ThermodynamicTemperature THotPin_internal
+    "Hot fluid temperature at pinch point, intermedaite variable";
+  Modelica.Units.SI.ThermodynamicTemperature THotOut_internal
+    "Hot fluid outgoing temperature, intermediate variable";
+  Modelica.Units.SI.HeatFlowRate QEva_flow_internal
+    "Evaporator heat flow rate, intermediate variable";
+
+equation
+
+  assert(not (TWorCon > TWorEva - 1 and ena),
+"*** In " + getInstanceName() +
+": Working fluid condensing temperature is too high and close to evaporating temperature.
+This is likely caused by the flow rate of cold fluid in the condenser being too low
+when the ORC is on.");
+
+  assert(not (pCon < 101325 - 1 and ena and useLowCondenserPressureWarning),
+"*** In " + getInstanceName() +
+": Working fluid condensing pressure is lower than 101325 Pa.
+If this is intended, set useLowCondenserPressureWarning = false to turn off this warning.",
+level=AssertionLevel.warning);
+
+  // Evaporator
+  QEva_flow = mHot_flow * cpHot * (THotOut - THotIn);
```

Comment:
If you simulate `Validation.VariableSource` in OMEdit, it will get a division by zero error and the debugger points to this equation. A solution would be to use the `ena` signal and if `ena=false`, set variables such as `QEva_flow` to zero (within an `if-then-else` block).

Reply:
Fixed in 9501acd and 38abd36.
This model now runs locally with omc and should pass CI.